autopts/bot/common.py

```diff
@@ -137,10 +137,6 @@ def run_test_cases(self):
 
```

Comment:
I would add also smth like this:
```suggestion

        # Ensure the default config has the lowest priority
        iut_config_items = list(self.iut_config.items())
        for i, value in enumerate(iut_config_items):
            if value[0] == self.config_default:
                iut_config_items.pop(i)
                iut_config_items.append(value)

```
Otherwise, we'd have to remember that the default configuration in the iut_config dictionary has to be at the very end.

Reply:
Can you elaborate a bit more please why should we have priorities between configs and why the order they run matters. I don't see any point in running tests with order. In my understanding and most of the use case default config is the base test list that should run. I don't see any gain in that.

Reply:
I see the point, will update the PR. Thanks a lot for the explanation.